pyotgw/pyotgw.py

```diff
@@ -493,18 +526,21 @@ def get_gpio_mode(self, gpio_id):
         if gpio_id in "AB" and mode in range(8):
             if mode == 7 and gpio_id != "B":
                 return None
-            cmd = globals().get("OTGW_CMD_GPIO_{}".format(gpio_id))
+            cmd = globals().get(f"v.OTGW_CMD_GPIO_{gpio_id}")
```

Comment:
Same

Reply:
Fixed in https://github.com/mvn23/pyotgw/pull/12/commits/f6ed560b474bbd63057d40757dc13a2988a73c72.